pkg/server/server.go

```diff
@@ -27,6 +28,15 @@ type JSONRPCServer struct {
 func (server JSONRPCServer) commandHandler(_ context.Context, reply jsonrpc2.Replier, req jsonrpc2.Request) error {
 	fmt.Println("Called method: " + req.Method())
 
+	defer func() {
+		err := recover()
+		rollbar.LogPanic(err, true)
+
+		if err != nil {
+			panic(err)
+		}
```

Comment:
It seems to me we'll _always_ report an error here, even if it's `nil`. Shouldn't `LogPanic` be inside the `if` too?

```suggestion
		err := recover()

		if err != nil {
			rollbar.LogPanic(err, true)
			panic(err)
		}
```